filter/useragent/filteruseragent.go

```diff
@@ -15,6 +15,7 @@ const ModuleName = "useragent"
 
 // errors
 var (
+	// Deprecated: this error is never returned.
```

Comment:
Not sure how concerned we are about backward compatibility and keeping this around. Happy to remove this if it's not an issue.

Reply:
I think it's OK to remove unused variables.